util/config/generate-license

```diff
@@ -9,6 +9,8 @@ parser.add_argument('destination', help='The location to write the updated LICEN
 
 args = parser.parse_args()
 
+out_file = os.path.join(args.destination, 'LICENSE')
```

Comment:
It looks like we can revert this file now? I don't mind the variable, but if we want to keep it let's move it closer to the `open` call.

Reply:
I just removed all these since they aren't necessary